plugins/remark-lint-heading-sentence-case/package.json

```diff
@@ -26,23 +27,24 @@
 	"dependencies": {
 		"debug": "^4.3.2",
 		"escape-string-regexp": "^4.0.0",
-		"unified-lint-rule": "^1.0.2",
-		"unist-util-position": "^3.1.0",
-		"unist-util-visit": "^2.0.0",
-		"vfile-location": "^3.0.1"
+		"unist-util-position": "^4.0.1",
+		"vfile-location": "^3.0.1",
+		"unified-lint-rule": "^2.1.0",
+		"unist-util-generated": "^2.0.0",
+		"unist-util-visit": "^4.1.0"
 	},
 	"devDependencies": {
 		"@semantic-release/changelog": "^5.0.1",
```

Comment:
**issue**: npm audit finds 6 moderate, 216 high issues with this module.  222 of the issues are related to **semantic-release**.  The package files in the plugins need to be clean because any issues found with open-docs are going to be public, and Ingo is checking them, so they will come back to haunt us.  If we're not using these, get rid of them, and delete the "release" chunk:

1. @semantic-release/changelog
2. @semantic-release/git
3. semantic-release
4. tape
5. unist-util-generated

Reply:
fixed